### PR TITLE
Changing downstream contact emails

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: ci/redmine_bugzilla.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 BUGZILLA_URL = "https://bugzilla.redhat.com"
 REDMINE_URL = "https://pulp.plan.io"
 
-DOWNSTREAM_CONTACTS = ["dkliban@redhat.com", "ttereshc@redhat.com"]
+DOWNSTREAM_CONTACTS = ["daviddavis@redhat.com", "bmbouter@redhat.com"]
```
Comment (daviddavis) [inline ci/redmine_bugzilla.py:38]: @dkliban @goosemania please let me know if you want to remain contacts.

@bmbouter please let me know if you are not ok with being emailed.

Comment (fao89) [reply ci/redmine_bugzilla.py:38]: it is hardcoded here also:
https://github.com/pulp/pulp-ci/blob/master/.github/send_email.py#L64-L70

Comment (daviddavis) [reply ci/redmine_bugzilla.py:38]: Should the send_email.py addresses be different than the ones here (ie do we imagine that the send_email addresses will be used for other jobs besides this script)?

Comment (fao89) [reply ci/redmine_bugzilla.py:38]: for now, it is only used by redmine CI job, it was introduced just for it.
It was thought for CI to use to deliver emails, maybe in the future, we will use it for other jobs

**Pull request closed without merging**